code/utils/calc_t.py

```diff
@@ -0,0 +1,23 @@
+
+import numpy.linalg as npl
+from scipy.stats import mstats
+from scipy.stats import t
+
+def significant(a,b,c):
+    df=len(b)-2
+    y_hat = a.dot(c)
+    residuals = b - y_hat
+    ma=npl.pinv(a.transpose().dot(a))
+    RSS = np.sum(residuals ** 2)
+    df = a.shape[0] - npl.matrix_rank(a)
+    MRSS = RSS / df
+    s2 = MRSS
+    v_cov = s2 * npl.inv(a.T.dot(a))
+    numerator1 = c[0]
+    denominator1 = np.sqrt(v_cov[0, 0])
```

Comment:
I think you need to import numpy as np, you used it here,

can delete your 'scipy.stats' imports, right?